lib/index.js

```diff
@@ -5,10 +5,11 @@
 'use strict';
 
 const formatter = require('./formatter');
+const flowId = process.env['JEST_TEAMCITY_FLOWID'] || process.pid.toString();
```

Comment:
It looks like it's not jest specific feature. Could you rename it to `TEAMCITY_FLOWID`?

Reply:
Done